command/genesis/params.go

```diff
@@ -93,7 +100,7 @@ func (p *genesisParams) validateFlags() error {
 	}
 
 	// Check that the epoch size is correct
-	if p.epochSize < 2 && server.ConsensusType(p.consensusRaw) == server.IBFTConsensus {
+	if p.epochSize < 2 && (p.isIBFTConsensus() || p.isPolyBFTConsensus()) {
```

Comment:
Why not extract this second part of the conditional into a separate method like `hasEpochs`, or something similar?